src/modules/tccpp/components/starboard.ts

```diff
@@ -107,6 +140,12 @@ export default class Starboard extends BotComponent {
     }
 
     override async on_ready() {
+        const state = await this.database.component_state.findOne({ id: "starboard" });
+        this.delete_emojis = state?.delete_emojis ?? [];
+        this.ignored_emojis = state?.ignored_emojis ?? [];
+        this.negative_emojis = state?.negative_emojis ?? [];
+        this.repost_emojis = state?.repost_emojis ?? [];
```

Comment:
If the component state doesn't exist, should we error or insert it?

Reply:
We could, I went this route because I thought it'd be slightly simpler and easier to maintain as there's only one place where state is written to the db.